src/constants/index.js

```diff
@@ -1,20 +1,19 @@
-
 const listOfParticipants = [
-    { index: 1, name: 'Alejandro', description: "La QC del equipo de UI" },
-    { index: 2, name: 'Andres', description: "Andresito" },
-    { index: 3, name: 'John', description: "La avestruz" },
-    { index: 4, name: 'Wilfer', description: "La persona más decente del POD" },
-    { index: 5, name: 'Nihtxofely', description: "Netflix, cheese trees, el papá le tenía rabia" },
-    { index: 6, name: 'Jose Alejandro', description: "Jiu jitsu boy" },
-    { index: 7, name: 'Josimar', description: "Las burras" },
-    { index: 8, name: 'Ximena', description: "La crocssss" },
-    { index: 9, name: 'Jose', description: "El chico futbolista" },
-    { index: 10, name: 'Astrid', description: "La nueva BA, no no no no, bueno si" },
-    { index: 11, name: 'Cristhian', description: "El super crack" },
-    { index: 12, name: 'Fabricio', description: "El piñin" },
-    { index: 13, name: 'Leo', description: "El chico automation" },
-    { index: 14, name: 'Nestor', description: "El mejor UX" },
-    { index: 15, name: 'Nacho', description: "El mejor VD" }
+    { index: 1, name: 'Alejandro', description: "Web UI Developer" },
```

Comment:
Formalizing stuff? 
Still, we can think in not uploading this in the future as sensitive information and let the upload handle it

Reply:
got it